admin/tabs/activity/tracking/charts/ChartsModel.js

```diff
@@ -0,0 +1,228 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2020 Extremely Heavy Industries Inc.
+ */
+import {cloneDeep, reverse} from 'lodash';
+import {HoistModel, managed, XH} from '@xh/hoist/core';
+import {bindable} from '@xh/hoist/mobx';
+import {ChartModel} from '@xh/hoist/cmp/chart';
+import {LocalDate} from '@xh/hoist/utils/datetime';
+import {chart} from '@xh/hoist/cmp/chart';
+import {fmtDate} from '@xh/hoist/format';
+import {Icon} from '@xh/hoist/icon';
+import {TabContainerModel} from '@xh/hoist/cmp/tab';
+
+@HoistModel
+export class ChartsModel {
+
+    @bindable.ref data;
+    @bindable.ref dimensions;
+    @bindable chartAllLogs = false;
+    @bindable enableTimeseries = true;
+
+    @managed
+    tabContainerModel = new TabContainerModel({
+        tabs: [
+            {id: 'Histogram', icon: Icon.chartBar(), content: () => chart({model: this.categoryChartModel})},
+            {id: 'Timeseries', icon: Icon.chartLine(), content: () => chart({model: this.timeSeriesChartModel})}
+        ]
+    })
+
+    @managed
+    timeSeriesChartModel = new ChartModel({
+        highchartsConfig: {
+            chart: {type: 'line'},
+            plotOptions: {
+                line: {animation: false}
+            },
+            title: {text: null},
+            xAxis: {
```

Comment:
think maybe we should add drag zoom capabilities? Or time period buttons (although the date filters in the grid above accomplish the same)? Thinking of client app that this will be put to use in first as a backdrop; the admins of that app will be used to a certain kind of (useful) behavior.

Reply:
Think we should keep it simple - want to get this out.